test-monitor/nitestmonitor.yml

```diff
@@ -0,0 +1,1277 @@
+      description: Delete Test Steps Partial Success Response
+      title: StepsDeletePartialSuccessResponse
+      type: object
+      properties:
+        steps:
+          description: An array of test step id result id paris that were deleted
+          type: array
+          items:
+            $ref: '#/definitions/StepIdResultIdPair'
+        failed:
+          description: An array of test step id result id paris that failed to delete
+          type: array
+          items:
+            $ref: '#/definitions/StepIdResultIdPair'
+        error:
+          $ref: '#/definitions/Error'
+paths:
+  /:
+    get:
+      operationId: RootEndpoint
+      x-ni-request-all-privileges: true
+      summary: API information
+      description: Returns information about API versions and available operations.
+      tags: [versioning]
+      security: []
+      responses:
+        200:
+          description: OK
+          schema:
+            description: Version information
+            title: RootEndpointResponse
+            type: object
+            properties:
+              v1:
+                $ref: '#/definitions/V1Operations'
+              version:
+                description: The implementation version of the web service
+                type: string
+  /{version}:
+    parameters:
+      - in: path
+        name: version
+        description: The version of the API to retrieve operations.
+        type: string
+        required: true
+    get:
+      operationId: RootEndpointWithVersion
+      x-ni-request-all-privileges: true
+      summary: API version information
+      description: Returns available operations for a single version of the API.
+      tags: [versioning]
+      security: []
+      responses:
+        200:
+          description: OK
+          schema:
+            $ref: '#/definitions/V1Operations'
+        404:
+          description: Not Found
+          schema:
+            $ref: '#/definitions/Error'
+  /v1/query-results:
+    post:
+      operationId: query-results
+      x-ni-operation: queryResults
+      x-ni-privilege: Read
+      summary: Queries test results
+      description: Queries test results
+      tags:
+        - results
+      parameters:
+        - in: body
+          name: postBody
+          description: A result query object
+          required: false
+          schema:
+            $ref: '#/definitions/TestResultQueryObject'
+        - in: query
+          name: skip
+          type: integer
+          default: 0
+        - in: query
+          name: take
+          type: integer
+          default: -1
+      responses:
+        200:
+          $ref: '#/responses/ResultsQueryResponse'
+        401:
+          $ref: '#/responses/Unauthorized'
+        default:
+          $ref: '#/responses/Error'
+  /v1/results:
+    put:
+      operationId: update-results
+      x-ni-operation: updateResults
+      x-ni-privilege: Write
+      summary: Update test results
+      description: Update test results
```

Comment:
Updates existing test results

Alternatively, you could delete the summary from all these routes, since they're usually the same as the description, and IIRC, @pvallone only used descriptions.